UnityProject/Assets/Scripts/UI/Systems/AdminTabButtons.cs

```diff
@@ -84,7 +84,6 @@ private void DisableAllGUI()
 			devCloner.gameObject.SetActive(false);
 			devDestroyer.gameObject.SetActive(false);
 			devSelectTile.gameObject.SetActive(false);
-			vv.SetActive(false);
```

Comment:
Sorry, can you explain what happened here?

Reply:
The vv script is actually on a sub object, so setting active breaks it.

I added that in the tile placer PR, as i thought it was just missing.